### ISSUE TITLE
Won't start from KDE launcher.

### ISSUE THREAD

**Issue by donhilion** [sedwards2009/extraterm#124]
I'm using extraterm in der Verion 0.35.
My system is Arch Linux with KDE/Plasma.

When I try to start extraterm from the launcher (either start menu or alt+f2) nothing happens.
Starting from another terminal or a launcher like cerebro works.

**Comment by sedwards2009** [comment]
How did you install Extraterm? Did you install from an arch repo or just download a zip file from github?

**Comment by sedwards2009** [comment]
@SolarAquarion do you know where the problem is here?

**Comment by Stele77** [comment]
I can confirm this.. But i switched to Linux only a short time ago, so i can not do much to actually find this bug.. Sorry..

**Comment by ChanceNCounter** [comment]
I realize this is a year-old issue, but it's still open and I'm having precisely the same problem, though not on Arch.

I'm on Ubuntu Disco (the default GNOME-based desktop.) I downloaded `extraterm-0.46.0-linux-x64.zip` and extracted it. I don't have Cerebro, but I've got uLauncher, which also can't seem to launch it.

Running extraterm from gnome-terminal doesn't do anything. Running it from uLauncher also doesn't do anything. Double-clicking it doesn't work because Gnome seems to think it's a shared library (though it's executable.)

Konsole, however, _can_ launch extraterm.
